fam_initialize.html

```diff
@@ -219,7 +226,7 @@ <h2 id="e1">Example</h2>
 	Fam_Options *fm = (Fam_Options*)malloc(sizeof(Fam_Options));
 	memset((void *)fm, 0, sizeof(Fam_Options));
```

Comment:
What does the user need to do to pick only some options to be default? The description above seems to be an "all or nothing" behavior.